angular.json

```
@@ -16,17 +20,12 @@
             "outputPath": "dist/movie-owt",
             "index": "src/index.html",
             "main": "src/main.ts",
-            "polyfills": [
-              "zone.js"
-            ],
+            "polyfills": ["zone.js"],
             "tsConfig": "tsconfig.app.json",
-            "assets": [
-              "src/favicon.ico",
-              "src/assets"
-            ],
+            "assets": ["src/favicon.ico", "src/assets"],
             "styles": [
               "@angular/material/prebuilt-themes/indigo-pink.css",
-              "src/styles.css"
+              "src/assets/scss/themes/theme.scss"
```

```suggestion
              "src/assets/scss/styles.scss"
```
trong styles.scss này e sẽ import "./theme/theme.scss"